eclass/flag-o-matic.eclass

```diff
@@ -876,4 +876,147 @@ no-as-needed() {
 	esac
 }
 
+# @FUNCTION: _test-compile-PROG
```

Comment:
Thanks for abstracting this. It needed to be done and it's been on the internal-to-my-brain TODO list for ages.